Makefile

```diff
@@ -0,0 +1,39 @@
+########################################################
+
+NAME := ftl_installer
+
+# VERSION file provides one place to update the software version.
+VERSION := $(shell cat VERSION)
+
+__init__.py: ftl_installer/__init__.py.in
+	sed "s/%VERSION%/$(VERSION)/" $< > ftl_installer/__init__.py
+
+setup.py: setup.py.in
+	sed "s/%VERSION%/$(VERSION)/" $< > setup.py
+
+venv: clean __init__.py setup.py
+	@echo "#############################################"
+	@echo "# Creating a virtualenv"
+	@echo "#############################################"
+	virtualenv venv
```

Comment:
Added an `FTL_PYTHON` variable to the Makefile ex: `FTL_PYTHON=/usr/bin/python3.4 make venv`. I also added a set of ci (`make ci`) make targets that operate within the virtualenv ex: `FTL_PYTHON=/usr/bin/python3.4 make ci`